Update heartslider.js goToSlide function

TeasingMeep opened the pull request:
Add check in goToSlide to ensure targetIndex is typeof number

TeasingMeep committed:
Update heartslider.js

Added check in goToSlide to ensure targetIndex is typeof number

austenhart reviewed (changes requested):
Let's use Number() instead of parseInt, and that check needs to be at the very beginning of the goToSlide func. The Number function will assign a value of 0 if the value is undefined or null, which is a perfect fallback for this.

TeasingMeep committed:
Update heartslider.js

Changed parseInt() to Number() and moved to very beginning of the goToSlide func.

TeasingMeep commented:
Sounds good, updated the branch to reflect comments made.
